CHANGELOG.md

```diff
@@ -68,6 +68,13 @@ https://github.com/googleapis/google-cloud-cpp/issues/8234.
 
 ## v1.38.0 - TBD
 
+### [Service Management API](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/servicemanagement/README.md)
+
+* The `DisableService()` and `EnableService()` RPCs are now retired.  These RPCs
+  were deprecated and non-functional at the time this client library was
+  generated.  We do not expect this will actually break any existing code.
+  Nevertheless, we apologize for any confusion.
```

Comment:
Perhaps this relates to a previous comment I made that never gained any traction?

https://github.com/googleapis/google-cloud-cpp/pull/8087#pullrequestreview-864052924

Reply:
I think that previous comment is valid.  In this case, the RPC was deprecated, using `option deprecated = true` and the generator looks at the `[deprecated = true]` annotation on message fields.  Filed #8486